dpti/workflows/simulations/result_base.py

```
@@ -0,0 +1,112 @@
+    # result_type: str = Field(..., description="Type of result (user-defined)")
+
+    # Status
+    completeness: ResultCompleteness = ResultCompleteness.COMPLETE
+    # experiment_type: ExperimentType = ExperimentType.SIMULATION
+
+    # Basic provenance
+    creator: str = Field(default="unknown")
+    # parent_uuid: Optional[UUID] = None
+
+    # Custom metadata (user-defined)
+    custom_meta: Dict[str, Any] = Field(default_factory=dict)
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)